apps/indexer-coordinator/package.json

```diff
@@ -72,13 +70,16 @@
     "semver": "^7.3.7",
     "typeorm": "0.2.40",
     "uuid": "^8.3.2",
-    "yargs": "^17.2.1"
+    "yargs": "^17.2.1",
+    "graphql": "15.7.2",
+    "@apollo/client": "~3.7.16",
+    "@subql/contract-sdk": "~0.13.5",
+    "react": "^18.2.0"
```

Comment:
react shouldn't be a dependency of coordinator. I guess it was introduced from `@apollo/client`, but if you only use `@apollo/client/core` then you won't need `react`